### PR TITLE
build: split the jest test to run in parallel

### PR DESCRIPTION
**Pull request by goller**
Signed-off-by: Chris Goller <goller@gmail.com>

Split the jest tests across 8 circleci containers.

It appears that we get a linear speedup as we increase the parallelism of the jest tests.
Sometimes jest tests were taking up to 30 minutes.  This should decrease that to around 4 minutes.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

**Comment by TCL735** [comment]
🎆

File: ui/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "test:ju:report": "junit-viewer --results=junit-results --save-file=cypress/site/junit-report.html",
     "test:ju:clean": "rm junit-results/*.xml",
     "test:ju:all": "yarn test:ju:clean && yarn test:junit; yarn test:ju:report;",
-    "test:ci": "JEST_JUNIT_OUTPUT_DIR=\"./coverage\" jest --ci --coverage --reporters=default --reporters=jest-junit",
+    "test:ci": "JEST_JUNIT_OUTPUT_DIR=\"./coverage\" jest --ci --coverage --maxWorkers=2",
```
Comment (drdelambre) [inline ui/package.json]: this run command was added specifically for making jenkins tests run good within the CI pipeline. can you make sure that's still the case with these changes?

Comment (goller) [reply ui/package.json]: Done!

File: ui/tsconfig.json [hunk 2]
```diff
@@ -24,7 +24,8 @@
     "checkJs": false,
     "sourceMap": true,
     "esModuleInterop": true,
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "resolveJsonModule": true
```
Comment (drdelambre) [inline ui/tsconfig.json]: like... shouldn't this have broken sooner?! the things that keep me up at night...

**Pull request merged**